editor/plugins/path_2d_editor_plugin.cpp

```diff
@@ -72,15 +72,15 @@ bool Path2DEditor::forward_gui_input(const Ref<InputEvent> &p_event) {
 
 	Ref<InputEventMouseButton> mb = p_event;
 
+	real_t grab_threshold = EDITOR_DEF("editors/poly_editor/point_grab_radius", 8);
```

Comment:
Would make more sense to have it before the definition of `mb` than between it and its validity check.
Also this should use `EDITOR_GET`, no need to attempt redefining it all the time (I know it's not your code, but while at it it would be good to fix it).

Other instances that could be fixed:
```
$ rg "editor/point_grab"
editor/editor_settings.cpp
496:    _initial_set("editors/poly_editor/point_grab_radius", 8);

editor/plugins/collision_polygon_editor_plugin.cpp
147:            real_t grab_threshold = EDITOR_DEF("editors/poly_editor/point_grab_radius", 8);

editor/plugins/canvas_item_editor_plugin.cpp
459:    const real_t grab_distance = EDITOR_DEF("editors/poly_editor/point_grab_radius", 8);

editor/plugins/path_2d_editor_plugin.cpp
82:             real_t grab_threshold = EDITOR_DEF("editors/poly_editor/point_grab_radius", 8);

editor/plugins/abstract_polygon_2d_editor.cpp
425:                                    const real_t grab_threshold = EDITOR_DEF("editors/poly_editor/point_grab_radius", 8);
692:    const real_t grab_threshold = EDITOR_DEF("editors/poly_editor/point_grab_radius", 8);
723:    const real_t grab_threshold = EDITOR_DEF("editors/poly_editor/point_grab_radius", 8);
```

(the `_initial_set` should stay as is of course, it's what defines it originally for use by all abstracted polygon editors)